Google.Cloud.EntityFrameworkCore.Spanner/Extensions/TracerProviderExtension.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright 2021, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using OpenTelemetry.Trace;
+
+namespace Google.Cloud.EntityFrameworkCore.Spanner.Extensions
+{
+    public class TracerProviderExtension
+    {
+        private TracerProviderExtension()
+        {
+        }
+
+        public static readonly string SPAN_NAME_COMMIT = "SpannerEFCore.commit";
+        public static readonly string SPAN_NAME_SAVECHANGES = "SpannerEFCore.savechanges";
```

Comment:
nit: In the Java client the span names are written in camel case, so it would then be `SaveChanges` (see https://github.com/googleapis/java-spanner/blob/39007f6f25e94ba740b4528feba947af4a47a267/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java#L60). Is there a different convention for C#? If not, could we change these to use camel case?

Reply:
Done!